corehq/apps/reports/tests/test_export_response.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime, timedelta
+
+from django.test import TestCase
+from django.test.client import RequestFactory
+from django.test.utils import override_settings
+from mock import MagicMock, patch
+
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.reports.standard.monitoring import WorkerActivityReport
+from corehq.apps.users.models import WebUser
+from dimagi.utils.dates import DateSpan
+
+
+@override_settings(CACHE_REPORTS=True)
+class ExportResponseTest(TestCase):
+    domain = "export-response"
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.domain_obj = create_domain(cls.domain)
+        cls.request_factory = RequestFactory()
+        cls.couch_user = WebUser.create(cls.domain, "export-response-test", "foobar", None, None)
+        cls.couch_user.add_domain_membership(cls.domain, is_admin=True)
+        cls.couch_user.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.couch_user.delete(deleted_by=None)
+        cls.domain_obj.delete()
+        super().tearDownClass()
+
+    def test_export_response_returns_200(self):
+        request = self.request_factory.post(f'/{self.domain}/reports/export/worker_activity')
```

Comment:
Should this have an `/a/` in front of it? Does the URL even matter for the test if you're not using the `client`?

Reply:
No. This URL will not matter. I have just added it for the context that we are using worker activity report. It can be any random string.